input/input.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://utilities91.netlify.app/styles.css" />
+    <link rel="stylesheet" href="./input.css" />

```

Comment:
In line 9 looks like you have imported css and i see you have imported it in polorid css also... is it intentional. If yes, please ignore my comment

Reply:
Forgot to remove that file :)